ocaml/xapi/quicktest_cbt.ml

```diff
@@ -123,7 +123,9 @@ let vdi_data_destroy_test ~session_id ~vDI =
       ~msg:"VDI.data_destroy failed to update VDI.type";
     assert_cbt_status true ~session_id ~test ~vDI:snapshot
       ~msg:"VDI snapshot cbt_enabled field erroneously set to false";
-    test_vdi_update ~session_id ~test snapshot;
+  (*  test_vdi_update ~session_id ~test snapshot; 
+      temporarily comment this out as it is blocked on CA-273981
+      VDI.update doesn't currently work on cbt-metadata VDIs *)
```

Comment:
Actually we can use `SR.scan` until CA-273981 is resolved, so I think we should put that below this comment to still get some testing.